samples/managed_subscribe/.accepted_words.txt

```diff
@@ -12,9 +12,11 @@ github
 http
 https
 Ibeji
+InVehicle
 invehicle
 MQTT
 repo
 svg
+TopicManagementCB
```

Comment:
What does `CB`  mean? Does it stand for callback?

Reply:
Correct, it stands for callback